.circleci/config.yml

```
@@ -108,6 +108,29 @@ jobs:
                 name: Tag
                 command: |
                   ./.circleci/ci-docker-tag-op-geth-release.sh <<parameters.registry>>/<<parameters.repo>> $CIRCLE_TAG $CIRCLE_SHA1
+      - when:
+          condition:
+            equal: [optimism, << pipeline.git.branch >>]
+          steps:
+            - gcp-oidc-authenticate:
+                service_account_email: GCP_SERVICE_ATTESTOR_ACCOUNT_EMAIL
+            - run:
+                name: Sign
+                command: |
+                  git clone https://github.com/ethereum-optimism/binary_signer
+                  cd binary_signer/signer
+                  git checkout tags/v1.0.3
```

```suggestion
                  git clone --branch v1.0.3 --depth 1 https://github.com/ethereum-optimism/binary_signer
                  cd binary_signer/signer
```
makes sure you clone directly to the needed tag, and don't download the whole git history. More lightweight.